app/src/androidTest/java/com/github/studydistractor/sdp/ui/EventTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.github.studydistractor.sdp.ui
+
+import androidx.compose.ui.test.assertHasClickAction
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertIsNotDisplayed
+import androidx.compose.ui.test.assertTextContains
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import com.github.studydistractor.sdp.event.EventViewModel
+import com.github.studydistractor.sdp.fakeServices.EventServiceFake
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class EventTest {
```

Comment:
It would be nice to add some more tests, e.g. to see if all components are displayed correctly, if buttons are clickable etc to increase our test coverage.

Reply:
You are right, I think I was looking at something else when I made the comment, sorry!